src/interface/apiResponse.ts

```diff
@@ -23,6 +23,11 @@ export class FolderResponse {
   path: string;
 }
 
+export class DeleteFolderResponse {
+  @ApiProperty()
+  parentPath: string;
```

Comment:
why parentPath and not just path or key?

Below are our internal agreements about naming.
path: etc/var/app
location: etc/var
name(for folder): app
key(for file): etc/var/app/ics.txt